style-guide.rst

```diff
@@ -0,0 +1,628 @@
+
+Semicolons can almost always be replaced with periods (full stops). This rarely diminishes correctness and often improves readability.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  These "canned phrases" are pervasive in technical writing; remove them whenever they occur.
+  
+.. rubric:: Better
+
+.. code-block:: rst
+
+  These "canned phrases" are pervasive in technical writing. Remove them whenever they occur.
+
+.. _pronouns:
+    
+Pronouns
+----------
+
+.. _third-person-pronouns:
+
+Third-person personal pronouns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Third-person personal pronouns are:
+
+- he/him/his
+- she/her/her(s)
+- they/them/their(s) 
+
+.. note:: 
+
+  While some people consider *they/them/their* to be non-standard (or "incorrect") as third-person singular, it has gained wide use as a gender-neutral or gender-ambigious alternative to *he* or *she*.
+
+There are two issues with personal pronouns:
+
+- gender bias
+- clarity
+
+To avoid gender bias, the third person gender-neutral *they/then/their(s)* is preferred over *he* or *she* pronouns when writing about abstract individuals.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  The enumerator uses his device.
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  The enumerator uses their device.
+
+
+Unfortunately, *they/them/their* is not a perfect solution. Since it is conventionally used as a plural pronoun, it can cause confusion.
+
+Therefore, avoid the use of personal pronouns whenever possible. They are often out of place in technical writing anyway. Rewriting passages to avoid personal pronouns often makes the writing more clear.
+
+.. rubric:: Correct
+
+.. code-block:: rst
+
+  When using Collect, first the enumerator opens the app on their device. Then they complete the survey.
+  
+.. rubric:: Improved
+
+.. code-block:: rst
+
+  To use Collect:
+  
+  - open the app
+  - complete the survey
+
+.. _same:  
+  
+"Same"
+~~~~~~~~~
+
+*Same*, when used as an impersonal pronoun, is non-standard in Modern American English. It should be avoided.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  ODK Collect is an Android app. The same can be used for...
+  
+.. rubric:: Right
+
+.. code-block:: rst
+
+  ODK Collect is an Android app. It can be used for...
+
+  .. OR THIS:
+  
+  ODK Collect is an Android app that is used to...
+  
+
+.. _titles-style-guide:  
+  
+Titles 
+------------
+
+.. _title-casing:
+
+Title case and sentence case
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Document titles should be in ``Title Case`` -- that is, all meaningful words are to be capitalized.
+
+Section titles should use ``Sentence case`` -- that is, only the first word should be capitalized, along with any proper nouns or other words usually capitalized in a sentence.
+
+.. _title-verb-forms:
+
+Verb forms
+-----------
+
+If a document or section describes a procedure that someone might do, use a gerund. Do not use the "How to..." construction.
```

Comment:
I had to look up gerund. Maybe add a link or an example?

Reply:
Also, why not "How to". I kinda like the phrase. And I think people might search for "How to"

Reply:
Maybe it would be better to not use the word "gerund" since that is sort of unhelpful jargon.

> Why not "How to"

This is a good question. My ready answers are:
 - it feels right
 - it has become the standard in the TW community
 - "how to" is redundant -- most of documentation is how to

there are probably better answers

Reply:
I asked around for why *not "how to"* has become standard. The most compelling reason is that it makes a document or table of contents harder to scan, because of the redundancy.